src/libraries/System.Runtime.Interop.JavaScript.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
`src/libraries` has no `.csproj` files for concrete libraries and we want to keep it that way.

If this is meant just for whitebox testing of the common internal interop pieces, a good place to put this would be somewhere under src\libraries\Common\tests.

Reply:
We'd like to eventually expose this as official API (not in net5 timeframe) and also use it as implementation library inside src/libraries.

Reply:
- The .csproj still should not be at `src/libraries/System.Runtime.Interop.JavaScript.csproj` with this plan. It would need to be `src/libraries/System.Runtime.Interop.JavaScript/src/System.Runtime.Interop.JavaScript.csproj`.

- If you create it now, it will look pretty weird (e.g. it will have no no public APIs - all APIs are internal) and will likely just cause confusion. We can always moves things around once we decide that we want to do this.

- I would expect the eventual public name to be `System.Runtime.InteropServices.JavaScript` to match the other similar libraries.

Reply:
Added this library and referenced it from System.Net.Http project